.github/ISSUE_TEMPLATE/bug-fix.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug fix
+about: Give us your problem, and your solution
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Self checklist**
+
+ - [ ] This issue hasn't been already created
+ - [ ] Everything is sound and clear, all sections are filled-up
+ - [ ] Relevant labels have been added
```

Comment:
This is a bit vague, keep in mind that bugs can be filed by external people as well.

Reply:
Is it better now? https://github.com/inmanta/inmanta/pull/2416/commits/5090b071922f322cee876f9d6288b7279514d3b2